beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -168,7 +168,6 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 		}
 		s.pendingQueueLock.Unlock()
 		err := errors.Errorf("unknown parent for block with slot %d and parent root %#x", blk.Block().Slot(), blk.Block().ParentRoot())
-		log.WithError(err).WithFields(getBlockFields(blk)).Debug("Could not process early block")
```

Comment:
Nice!

Reply:
What would you recommend? Something like "Could not identify parent for block"?

Reply:
Sounds good to me